skycoin/skywire
# Use repo skywire-utilities

Did you run `make format && make check`?

Fixes #1117 

 Changes:	
- Used packages from `skywire-utilities` instead of `dmsg`

How to test this PR:
1. Checkout [#dmsg-163](https://github.com/skycoin/dmsg/pull/163)
2. Uncomment `replace github.com/skycoin/dmsg => ../dmsg` in go mod and run `makedep; make build`
3. Start visor `./skywire-visor -c skywire-config.json`
4. Check if everything works

Needs:
[#dmsg-163](https://github.com/skycoin/dmsg/pull/163)
